src/shared/components/post/post-listing.tsx

```diff
@@ -106,6 +106,7 @@ interface PostListingProps {
   siteLanguages: number[];
   showCommunity?: boolean;
   showBody?: boolean;
+  smallAvatars?: boolean;
```

Comment:
Do you have more changes you forgot to push? `smallAvatar` is defined as a prop but not passed in anywhere. I'm assuming there are also supposed to be changes to person and community listings that this prop gets passed down to.

Reply:
Missed that one, thanks. Removed and pushed.